### ISSUE TITLE
Not working Xubuntu 20.04

### ISSUE THREAD

**Issue by rafo-br** [H-M-H/Weylus#24]
I have tried before installing Weylus_0.7.2_amd64.deb and I couldn't get it to work. I'm trying now the new 0.8.0 version, but still no luck. I'm running Xubuntu 20.04:
`Linux version 5.4.0-40-generic (buildd@lcy01-amd64-011) (gcc version 9.3.0 (Ubuntu 9.3.0-10ubuntu2)) #44-Ubuntu SMP Tue Jun 23 00:01:04 UTC 2020`

Bellow the steps I've followed. They were pretty much the instructions found on Weylus project Main Page.
`sudo apt install ./Weylus_0.8.0_amd64.deb`

Output:
`Reading package lists... Done`
`Building dependency tree       `
`Reading state information... Done`
`Note, selecting 'weylus' instead of './Weylus_0.8.0_amd64.deb'`
`The following NEW packages will be installed:`
`  weylus`
`0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.`
`Need to get 0 B/7.897 kB of archives.`
`After this operation, 24,0 MB of additional disk space will be used.`
`Get:1 /home/rafobr/Downloads/Weylus_0.8.0_amd64.deb weylus amd64 0.8.0 [7.897 kB]`
`Selecting previously unselected package weylus.`
`(Reading database ... 215475 files and directories currently installed.)`
`Preparing to unpack .../Weylus_0.8.0_amd64.deb ...`
`Unpacking weylus (0.8.0) ...`
`Setting up weylus (0.8.0) ...`

Then I set
`sudo groupadd -r uinput`
`sudo usermod -aG uinput $USER`
`echo 'KERNEL=="uinput", MODE="0660", GROUP="uinput", OPTIONS+="static_node=uinput"' | sudo tee /etc/udev/rules.d/60-weylus.rules`

No error message. I checked there is a 60-weylus.rules file and the content is
`'KERNEL=="uinput", MODE="0660", GROUP="uinput", OPTIONS+="static_node=uinput"'`

Then I ran
`sudo udevadm control --reload`
`sudo udevadm trigger`

Logged out and logged in again.
Opened weylus, pressed start. Log says:
`INFO Webserver listening at 0.0.0.0:1701...`

However, I got no luck accessing via Safari on my Ipad (iPadOS). The request to open the page timed out. 
I found strange the log said listening at `0.0.0.0:1701` . So I've tried changing the bind address to 192.168.0.100.  Log then said:
`INFO Webserver listening at 192.168.0.100:1701...`

But there was no luck either. What is the purpose of the _bind adress_ by the way?
I did not install the intel driver for screen mirroring. I haven't found it. Is that required for just using as a stylus table?
In summary, what am I missing here? Thanks for your help!

**Comment by H-M-H** [comment]
> However, I got no luck accessing via Safari on my Ipad (iPadOS). The request to open the page timed out.

This sounds like a network problem rather than a problem with Weylus itself. Please check if you can connect to Weylus from your computer. If this works something is blocking the connection between your computer and iPad. Maybe you have some kind of firewall setup, is `ufw` installed? Or maybe the iPad and your computer are on different networks.

> What is the purpose of the bind adress by the way?

Typically your computer has multiple IP addresses assigned, a local one (127.0.0.1) that is only reachable from your computer itself and in your case another IPv4 address 192.168.0.100. But there can be more e.g. an IPv6 address or another IPv4 address. Bind address specifies which address Weylus should use to serve the webpage and 0.0.0.0 is an alias for all IPv4 addresses available.

> I did not install the intel driver for screen mirroring. I haven't found it. Is that required for just using as a stylus table?

No, this is not required.

**Comment by H-M-H** [comment]
I assume this is solved now, if not feel free to reopen this issue.
